app/dolphin/lib/common.js

```diff
@@ -30,7 +30,7 @@ function DolphinOnCreateUser(options, user) {
 
 if (Meteor.isServer) {
 	Meteor.startup(() =>
-		settings.get('Accounts_OAuth_Dolphin_URL', (key, value) => {
+		settings.watch('Accounts_OAuth_Dolphin_URL', (key, value) => {
```

Comment:
`settings.watch` callback receives only the value:
```suggestion
		settings.watch('Accounts_OAuth_Dolphin_URL', (value) => {
```